app/models/option_select_facet.rb

```diff
@@ -42,10 +42,6 @@ def has_filters?
     selected_values.any?
   end
 
-  def close_facet?
```

Comment:
I think we might need to keep this, so that facets that contain more than 10 items (i.e. organisations) are closed by default.

Reply:
Please see https://github.com/alphagov/finder-frontend/pull/1073/files#r279796574